src/main/java/com/github/instagram4j/instagram4j/requests/search/SearchUserRequest.java

```diff
@@ -0,0 +1,61 @@
+package com.github.instagram4j.instagram4j.requests.search;
+
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.requests.IGGetRequest;
+import com.github.instagram4j.instagram4j.requests.IGPaginatedRequest;
+import com.github.instagram4j.instagram4j.responses.search.SearchUserResponse;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.Optional;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class SearchUserRequest extends IGGetRequest<SearchUserResponse> {
```

Comment:
This request has already been implemented in #531 . Please remove this request and associated response.